getlantern.org/static/js/download.js

```diff
@@ -0,0 +1,52 @@
+$(document).ready(function(){
+
+  var ANDROID_LINK = "lantern-installer-beta.apk";
+
+  var set_download_link = function() {
+    var os = platform.os.architecture + platform.os.family;
+    var os_links = [
+      {regexp: "[2-6]{2}[Ww]indows", link: "lantern-installer-beta.exe"},
+      {regexp: "[2-6]{2}OS X", link: "lantern-installer-beta.dmg"},
+      // for Android devices with large screen
+      {regexp: ".*Android", link: ANDROID_LINK},
+      {regexp: "32.*", link: "lantern-installer-beta-32-bit.deb"},
+      {regexp: "64.*", link: "lantern-installer-beta-64-bit.deb"},
+      {regexp: ".*", link: "https://github.com/getlantern/lantern#lantern-"}
+    ];
+    for (i = 0; i < os_links.length; ++i) {
+      re = new RegExp(os_links[i].regexp);
+      if (os.match(re)) {
+        $("#download-button").attr("href", os_links[i].link);
+        break;
+      }
+    }
+
+    $('#download-button').on('click', function() {
+      ga('send', 'event', 'button', 'click', 'download');
+    });
+  };
+
+  var prepare_android_download = function() {
+    $("#download-android-button").attr("href", ANDROID_LINK);
+    if (platform.os.family === "Android") {
+      var downloadBtn = document.getElementById("download-button");
+      $("#download-android").css("display", "block");
+      $("#other-systems").css("display", "block");
+      $("#download-android-button").after(downloadBtn);
+    }
+  };
+
+  var update_version_number = function() {
```

Comment:
This fetches the version number and shows it. It doesn't affect `index.html` as relevant elements are absent there.